build/root/Makefile

```diff
@@ -311,16 +311,6 @@ clean:
 	hack/make-rules/clean.sh
 endif
 
-
-# TODO(thockin): Remove this in v1.29.
-.PHONY: generated_files
-generated_files:
-	echo "'make generated_files' is deprecated.  Please use hack/update-codegen.sh instead."
```

Comment:
well, this target is removed, is there anywhere in doc to introduce where to find the script/how to generate the code?